docker/requirements.txt

```diff
@@ -1,116 +1,100 @@
 # misc
-aiobotocore[boto3,awscli]==1.3.0
-docutils==0.15.2 # because of botocore
-pyyaml==5.4.1
-urllib3==1.26.5 # for botocore, but conflicts with pysal, which can not be updated as it dropped 3.6
-python-dateutil==2.8.0  # pinned because of pysal==2.2.0
-colorama==0.4.3
+access
+affine
+aiobotocore
+aiohttp
+alabaster
+colorama
+dill
+docutils
 cython
+ffmpeg-python
+param
+pydash
+pyyaml
+pyrsistent
+python-dateutil
 ruamel.yaml
 ruamel.yaml.clib
-pyrsistent
-param
-dill
-ffmpeg-python
+s3fs
 tqdm
+urllib3
 voluptuous
-pydash
-s3fs
 # pre-install misc datacube dependencies
-jsonschema
+bottleneck
 ciso8601
+jsonschema
+lark-parser
 psycopg2
+setuptools-scm
+setuptools-scm
 sqlalchemy
 toolz
-lark-parser
-setuptools_scm[toml]
-bottleneck
 # pre-install misc odc-tools dependencies
-lmdb
-zstandard
-thredds-crawler
 astropy
 boltons
 dataclasses
+lmdb
+thredds-crawler
+zstandard
 # pre-install eodatasets3 dependencies
 cattrs
 structlog
 
 # Scientific Stack
-dask[complete]==2.30.0
-distributed==2.30.0
+dask
+distributed
 dask-image
 dask-ml
-tensorflow==2.4.2
-scipy
-xgboost       # used by Dale's s2cloudmask
+msgpack
+numba
+numexpr
+numexpr3
+pandas
 pathos
+Pillow
 scikit-image
 scikit-learn
-msgpack==1.0.0
-numexpr
-numexpr3==3.0.1a1
-numba
+scipy
 statsmodels
-Pillow
-pandas==1.1.2
-xarray==0.16.1
-numpy
+tensorflow
+xgboost       # used by Dale's s2cloudmask
+xarray
 zarr
 ### plotting
-bokeh==2.1.1
-matplotlib==3.2.1
+bokeh
 descartes
+matplotlib
 seaborn
-https://dl.google.com/coral/python/tflite_runtime-2.1.0.post1-cp36-cp36m-linux_x86_64.whl
 
 # Geo stack
-geojson
-geopy
+cligj
 fiona
-rasterstats
+geohash2
+geojson
 geopandas
-cligj
+geopy
 GDAL
 rasterio
+rasterstats
 rio-cogeo
 Shapely
 pyproj
 h5py
 netcdf4
 rtree
+owslib
 pandana
-pysal==2.3.0
 pyepsg
-mapclassify
-urbanaccess
+pysal
 pysheds
-https://github.com/ubarsc/rios/archive/rios-1.4.10.tar.gz
-owslib
-rioxarray
 python-geohash
-geohash2
-richdem
 cmocean
+mapclassify
+rioxarray
+richdem
+urbanaccess
 
-
---no-binary=aiohttp\
-,cffi\
-,cftime\
-,cryptography\
-,lxml\
-,lmdb\
-,numexpr\
-,numexpr3\
-,protobuf\
-,pyzmq\
-,regex\
-,ruamel.yaml.clib\
-,yarl\
-,msgpack\
-,multidict\
-,statsmodels\
-,typed_ast\
-,tables\
-,zstandard\
-,richdem\
+https://github.com/ubarsc/rios/archive/rios-1.4.10.tar.gz
+# TODO - ?
+# https://dl.google.com/coral/python/tflite_runtime-2.1.0.post1-cp36-cp36m-linux_x86_64.whl
```

Comment:
this also didn't work

Reply:
installed libcgal-dev and libboost-dev instead

Reply:
this is also expected, new image has newer python so you'll need to use `-cp38` version of tflite

Reply:
same story with `rsgislib` I compiled it ages ago for py36 only, it won't work with py38

Reply:
did this
```
--extra-index-url="https://google-coral.github.io/py-repo/"
tflite_runtime
```